src/notifiers/ntfy.py

```diff
@@ -0,0 +1,70 @@
+import json
+import logging
+
+import requests
+
+from models import Config, Item
+from models.errors import MaskConfigurationError, NtfyConfigurationError
+from notifiers import Notifier
+
+log = logging.getLogger('tgtg')
+
+
+class Ntfy(Notifier):
```

Comment:
The notifier could also be based on the `Webhook` notifier like the `IFTTT` notifier. It would only be necessary to add the `auth` property to the `Webhook` notifier, which would in general be a good addition.
This would centralize the request logic and make maintaining the code easier.
Using `self.__class__.__name__` can be used in the `Webhook` logs to display the calling Notifier.

Reply:
Good idea, but I think adding authentication to the `WebHook` should be an other pull request, because it is not part of this feature and affects `WebHook` and `IFTTT` functionality. But I like the idea to centralize the request logic and will do that. So are you fine with this argumentation?